### ISSUE TITLE
Code for PointNet++ 

### ISSUE THREAD

**Issue by barzanisar** [PRBonn/segcontrast#15]
Hey, 

In your paper you mentioned finetuning pointrcnn for 3D object detection. PointRCNN uses pointnet++ backbone but your code only has unet. How did you finetune pointrcnn with a unet backbone? Or do you plan to upload the code with pointnet++ ? Also, what are the hyperparameters used for finetuning pointrcnn on 5% Kitti? For example, epochs, the optimizer, learning rate, weight decay and lr scheduler etc.

Thanks!

**Comment by nuneslu** [comment]
Hi, we have used the [OpenPCDet](https://github.com/zaiweizhang/OpenPCDet) repo to run everything. This is the toolbox also used in one of the baselines, so we just made some adaptations to extract the point-wise features using the MinkUNet backbone instead of the PointNet++. You can check the code for object detection in [this](https://hub.docker.com/r/nuneslu/segcontrast_openpcdet) docker image. Inside the image, you will find the subsets of training and the hyperparameters for fine-tuning.

**Comment by barzanisar** [comment]
Hey, 

Thank you for replying. 
I have 2 more questions:

- For training on 5% kitti, do you use the dbinfos from the 100% train set or do you create dbinfos from 5% kitti split and only use that. For example, for kitti_infos_train_5_0.pkl, did you create kitti_dbinfos_train_5_0.pkl or used the kitti_dbinfos_train.pkl?
- From your image, I see that in segcontrast_pretrained_0p05_0.yaml, number of epochs is set to 80. Does this mean that in your paper the results for finetuning on 5% kitti are for 80 epochs?

Thank you!

**Comment by nuneslu** [comment]
- The ```dbinfo``` file is the same. We just use the ```kitti_infos_train_5_0.pkl``` that is on the docker image.
- Yes, for all the training subsets we use 80 epochs. In our paper we just increase the number of epochs for few training data for the semantic segmentation.
